### ISSUE TITLE
Dark theme doesn't work on Win 11

### ISSUE THREAD

**Issue by gerroon** [ransome1/sleek#743]
### Important ###
Please follow this template strictly when reporting bugs. 

Reports that do not adhere to this template will not be addressed and will be closed.

**App Version:** Enter the version number, e.g., 1.2.3
2.0.14

**Platform:** Windows / macOS / Linux
Win 11

**Installation Method:** App Store / Direct Download / Other
Zip

**Expected Behavior:**
The app should use dark theme

**Actual Behavior:**
It is using light theme

**Steps to Reproduce:**
settings
set to dark theme
restart

**Screenshots:**
![image](https://github.com/user-attachments/assets/9411db0d-6b99-4062-996c-dd1591059702)

**Comment by ransome1** [comment]
@gerroon @tvmole @Gitofthelawn can you please check if this still happens in 2.0.16 or the latest pre-release? https://github.com/ransome1/sleek/releases/tag/v2.0.17-rc.4

**Comment by gerroon** [comment]
I still have the issue,  with 2.0.17, even after deleting the user data folder. I use some other electron based apps, and they all respect the dark theme preference.

**Comment by YU-7** [comment]
I don't have this problem and the version of sleek is 2.0.16. my os is win 11, i download the exe not zip.![Image](https://github.com/user-attachments/assets/15aa26b8-ee98-4c9b-8be4-7ad223b7104e)

![Image](https://github.com/user-attachments/assets/ed21489c-50f5-42db-848c-ba747ddbcc4e)

**Comment by ransome1** [comment]
> I still have the issue with 2.0.17, even after deleting the user data folder. I use some other electron based apps, and they all respect the dark theme preference even without explicitly telling the app to use dark theme.
> 
> I also use Sleek on a Win10 PC, that one respects the dark theme setting. So it seems to be a Win 11 thing.

@gerroon I now do have access to a Windows 11 virtual machine. Unfortunately it is a development version which does not allow to switch to dark mode. So I need you to help me out again here. I have done some refactoring and I hope it solves the issue you're having. You can download the latest version here: https://github.com/ransome1/sleek/releases/tag/v2.0.19

If sleek's theme setting is set to system, it automatically adjusts to the OS theme. If you choose either dark or light theme, it persists to either of it. If you are experiencing anything else the bug is still present.

**Comment by DamirTD** [comment]
> [@DamirTD](https://github.com/DamirTD)
> 
> > it seems this might be related to the first launch
> > Can I ask if with first launch you mean the literal first launch of the app, without it being installed previously, or the first launch after an update when the app was already installed?
> 
> And does it also happen with one of the portable versions (end with -Setup.exe)? https://github.com/ransome1/sleek/releases/tag/v2.0.19

Yes, it was the very first installation of the app. The app wasn’t installed before. That’s when the bug happened. I clicked back and forth a couple of times between the light and dark themes, and nothing changed. That’s when I restarted the app, and everything started working fine. Tonight, I’ll try installing the portable version as well as the Microsoft Store version again and see if the bug happens again.  Unfortunately, I don’t remember exactly when I installed it, maybe week or 2 weeks ago. It might indeed have been an older version that was installed

**Comment by DamirTD** [comment]
> [@DamirTD](https://github.com/DamirTD)
> 
> > it seems this might be related to the first launch
> > Can I ask if with first launch you mean the literal first launch of the app, without it being installed previously, or the first launch after an update when the app was already installed?
> 
> And does it also happen with one of the portable versions (end with -Setup.exe)? https://github.com/ransome1/sleek/releases/tag/v2.0.19

I completely removed Sleek and installed the version sleek-2.0.19-win-Setup.exe (220 MB, March 6), and I tried changing the theme to dark on the first launch. It didn't switch right away, but after clicking a few times, it worked. Heres the video, i uploaded it to youtube. [Watch video](https://youtu.be/hMtED2MhrIg)

**Comment by kusiak** [comment]
Installed sleek off latest release, setup exe file, no dark theme, nothing happens when switching themes. 
Uninstalled it and installed it from the store, same thing.

Console reports some cookie issues, unure if that is helpful, attached.

The html seems to be in light theme, when I edit the class to be dark it switches the side bar

<img width="888" height="609" alt="Image" src="https://github.com/user-attachments/assets/c234df90-1c47-434b-99ee-b5f48f3aa835" />

[console.log](https://github.com/user-attachments/files/22658095/console.log)

**Comment by nonplayer** [comment]
I had this problem today installing fresh on Win11, using the Windows Store release. Nothing recommended above worked, even after multiple uninstalls and reinstalls and change attempts, it remained in Light mode.

I removed it, then downloaded the latest EXE installer here, and the first time I tried it still wouldn't change. But after two more attempts to toggle the theme in the settings, finally it changed to dark mode.

**Comment by lunule** [comment]
> (...) downloaded the latest EXE installer here, and the first time I tried it still wouldn't change. But after two more attempts to toggle the theme in the settings, finally it changed to dark mode.

Just a confirmation: I just had the same experience today (2026-02-02, Windows 11) described above @nonplayer.

**Comment by ransome1** [comment]
@lunule @gerroon @nonplayer @kusiak @DamirTD @YU-7 I might have found the cause here, but I cannot test it because I don't have access to a fully unlocked Windows 10 or 11.

This fix depends on you guys deleting the existing config files before starting sleek: `C:\Users\[YOUR_USERNAME]\AppData\Roaming\sleek\userData`

The fix is included in the latest pre-release: https://github.com/ransome1/sleek/releases/tag/v2.0.25-rc.3

**Comment by DamirTD** [comment]
> [@lunule](https://github.com/lunule) [@gerroon](https://github.com/gerroon) [@nonplayer](https://github.com/nonplayer) [@kusiak](https://github.com/kusiak) [@DamirTD](https://github.com/DamirTD) [@YU-7](https://github.com/YU-7) I might have found the cause here, but I cannot test it because I don't have access to a fully unlocked Windows 10 or 11.
> 
> This fix depends on you guys deleting the existing config files before starting sleek: `C:\Users\[YOUR_USERNAME]\AppData\Roaming\sleek\userData`
> 
> The fix is included in the latest pre-release: https://github.com/ransome1/sleek/releases/tag/v2.0.25-rc.3

I’ve installed the latest pre-release (v2.0.25-rc.3-win-x64-Setup), and everything worked fine for me without any issues. iam using win 11

**Comment by ransome1** [comment]
Released as 2.0.25: https://github.com/ransome1/sleek/releases/tag/v2.0.25
